fluff/__init__.py

```diff
@@ -15,6 +15,9 @@
 import fluff.util
 
 
+default_app_config = 'fluff.app_config.FluffAppConfig'
```

Comment:
neat (had to go read [the docs](https://docs.djangoproject.com/en/1.8/ref/applications/#configuring-applications) to figure out what this was doing)